### ISSUE TITLE
Skin overlay not loading

### ISSUE THREAD

**Issue by simnights** [thepancake1/TS4-SimRipper#5]
Hello, 
Thanks so much for all your work on the new update. Since the newest version was fixed, I've noticed my sim's skin overlay is not loading. He has tattoos that I have added to the overlay, but they are missing in both the ripper preview and his texture files. The file is not listed in his information either. I've attached a picture of the skin overlay in the game for reference, as well as a screenshot of the ripper and a text file of the sim information. The overlay file - named LAZ SKIN OVERLAY.package is missing from the CC list. The overlay did work in the old version of the Ripper before the update. 

![16-02-2022 23-02-46](https://user-images.githubusercontent.com/99772892/154403077-c2d46dec-fb5f-4d4f-b8a9-ad50064bf6ce.png)
![02-13-22_11-14-28 PM](https://user-images.githubusercontent.com/99772892/154403099-ab9edc25-566e-4a53-8c4e-0784b3daf0bd.png)
[Errors.txt](https://github.com/thepancake1/TS4-SimRipper/files/8085452/Errors.txt)

**Comment by thepancake1** [comment]
Can you Save Sim Troubleshooting info and send it here?

**Comment by simnights** [comment]
[Uploading LazarusValentine_TroubleShoot.zip…]()

Of course.

**Comment by simnights** [comment]
[LazarusValentine_TroubleShoot.zip](https://github.com/thepancake1/TS4-SimRipper/files/8089104/LazarusValentine_TroubleShoot.zip)

**Comment by Shiningplumbob** [comment]
Hey I am actually having the same issue if anyone can help me the skin overlay does not load in ripper but does in game

**Comment by thepancake1** [comment]
> Hey I am actually having the same issue if anyone can help me the skin overlay does not load in ripper but does in game

Can you upload pictures of your sim as well as your SimRipper throubleshooting packages? As well as the error log,thanks.

**Comment by thepancake1** [comment]
> Hello, Thanks so much for all your work on the new update. Since the newest version was fixed, I've noticed my sim's skin overlay is not loading. He has tattoos that I have added to the overlay, but they are missing in both the ripper preview and his texture files. The file is not listed in his information either. I've attached a picture of the skin overlay in the game for reference, as well as a screenshot of the ripper and a text file of the sim information. The overlay file - named LAZ SKIN OVERLAY.package is missing from the CC list. The overlay did work in the old version of the Ripper before the update.
> 
> ![16-02-2022 23-02-46](https://user-images.githubusercontent.com/99772892/154403077-c2d46dec-fb5f-4d4f-b8a9-ad50064bf6ce.png) ![02-13-22_11-14-28 PM](https://user-images.githubusercontent.com/99772892/154403099-ab9edc25-566e-4a53-8c4e-0784b3daf0bd.png) [Errors.txt](https://github.com/thepancake1/TS4-SimRipper/files/8085452/Errors.txt)

Sorry for forgetting to reply to this. It's like, SimRipper just didn't recognize the package.
Can you answer these questions?
-  Did you put it in a subfolder?
- Is it an override?

**Comment by simnights** [comment]
No, it's in the main mods folder. I'm not sure what you mean by override. It's just a piece of CC that is marked as a skin detail. 

I actually noticed that his freckles were loading, but not the other skin detail (with the overlay and tattoos). So I ended up just adding the freckles to the overlay in Photoshop and then making the whole thing a skin detail marked as freckles, and that made it work for me. It's a hassle though.

**Comment by missmareaunabhabie** [comment]
I have the same problem My skin overlay does not show up in ripper along with some other skin details such as eyebags
